src/pathfinding_igvc/include/PathFinderNode.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Created By: Min Gyo Kim
+ * Created On: May 23rd 2018
+ * Description: Path Finder Node - subscribes to occupancy grid
+ * (nav_msgs/OccupancyGrid)
+ * and goal point (geometry_msgs/Point), obtains starting point from tf tree,
+ * and publishes the shortest path (nav_msgs/Path) from starting point to goal
+ * point.
+ */
+
+#ifndef PROJECT_PATHFINDERNODE_H
+#define PROJECT_PATHFINDERNODE_H
+
+#include <PathFinder.h>
+#include <nav_msgs/OccupancyGrid.h>
+#include <nav_msgs/Path.h>
+#include <ros/console.h>
+#include <ros/ros.h>
+#include <sb_utils.h>
+#include <tf/transform_listener.h>
+
+class PathFinderNode {
+    ros::Subscriber grid_subscriber;
+    ros::Subscriber goal_subscriber;
+    ros::Publisher publisher;
+
+    tf::TransformListener* _listener;
+
+    std::string _global_frame_name;
+    std::string _base_frame_name;
+
+    geometry_msgs::Point _goal;
+    nav_msgs::OccupancyGrid _grid;
+
+    bool _received_goal  = false;
+    bool _receivied_grid = false;
```

Comment:
Just for consistency, we should probably put these under the `private:` line, which makes the fact that they're private more explicit.